fluid/DeepASR/data_utils/data_reader.py

```diff
@@ -61,10 +61,21 @@ class SampleInfoBucket(object):
         label_bin_paths (list|tuple): Files containing the binary label data.
         label_desc_paths (list|tuple): Files containing the description of
                                        samples' label data.
+        split_perturb(int): split long sentence' perturb sub-sentence length value. 
```

Comment:
split_perturb(int) --> split_perturb (int)

Reply:
`split long sentence' perturb sub-sentence length value.` Please refine this comment.

Reply:
Random perturb sub-sentence length when split long sentence

Reply:
How about `Maximum perturbation value for length of sub-sentence when splitting long sentence.`?

Reply:
nice